dora/tests/integration/src/test/java/alluxio/client/cli/fs/command/DoraMkdirCommandIntegrationTest.java

```diff
@@ -51,7 +51,7 @@ public void testMkdirExisting() {
     assertEquals(0, mFsShell.run("mkdir", uri.toString()));
     assertEquals(-1, mFsShell.run("mkdir", uri.toString()));
     assertTrue(
-        mOutput.toString().contains("UNKNOWN: alluxio.exception.FileAlreadyExistsException"));
+        mOutput.toString().contains("ALREADY_EXISTS"));
```

Comment:
If we modify the code above, the output will be something like this: `ALREADY_EXISTS: /var/folders/pc/cpgks26x1lvcm03rzrmz06dh0000gn/T/junit9989567262251760026 already exists
`, it won't be a UNKNOWN Exception, so we have to do this change.